card/mod/standard/set/all/rich_html/editing.rb

```diff
@@ -139,7 +139,7 @@ def edit_type_buttons
   view :edit_structure, cache: :never do
     return unless card.structure
     voo.show :toolbar
-    render_related items: { view: :edit, nest_name: card.structure_card.name }
+    render_related items: { view: :edit, nest_name: card.structure_rule_card.name }
```

Comment:
edit > structure busted without this.  I'll try to write quick tests for this and the next one, but I mostly just fixed it so that I could get something done.  (it's another one where we're planning to replace the UI, but maybe not quite as soon as with bookmarking)

Reply:
(tests written)